magicctapipe/scripts/lst1_magic/lst1_magic_dl1_stereo_to_dl2.py

```diff
@@ -66,17 +67,16 @@ def apply_rfs(event_data, estimator):
 
     tel_ids = list(estimator.telescope_rfs.keys())
 
-    # Extract the events of the same telescope combination type
-    combo_type = list(TEL_COMBINATIONS.values()).index(tel_ids)
-    df_events = event_data.query(f"combo_type == {combo_type}")
+    # Extract the events with the same telescope ID
+    df_events = event_data.query(f"tel_id == {tel_ids[0]}")
```

Comment:
I do not understand why do you always take the first tel_id here

Reply:
Because the tel_ids list (i.e., the dictionary in 'estimator') only contains one element (we create one RF file per telescope and this 'list' will contain the ID of the telescope corresponding to the used RF file). So we take always the first because ti is the only tel_id